packages/frontend/src/components/table/filters/types.ts

```diff
@@ -0,0 +1,12 @@
+import type { filterIdToLabel } from './utils/labels'
+
+export interface FilterableEntry {
+  filterable: FilterableValue[] | undefined
+}
+
+export type FilterableValue = {
+  id: FilterableValueId
+  value: string
+}
+
+export type FilterableValueId = keyof typeof filterIdToLabel
```

Comment:
I would combine utils/labels, utils/filter-values and types into a single FilterableValue file